### ISSUE TITLE
The learning rate is different from the paper in iclr 2019

### ISSUE THREAD

**Issue by MrDavidG** [google-research/lottery-ticket-hypothesis#15]
Hi, thanks for sharing your code, and LTH is an interesting hypothesis. 

And I notice that the learning rate for mnist & lenet is different from the original paper, which is 1.2e-3. 
I'm looking for the settings of hyper parameters in LTH experiments, so I wonder whether the hyper-parameters in this repository are the same with those in the published paper. For example, the learning rate and the number of pruning iterations (whose defaults is 30 in this repository).
